07.high-fidelity-commerce/04.faq-section/01.commerce-faq/docs.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Commerce FAQs"
+taxonomy:
+    category:
+         docs
+---
+<a id="top"></a>
+
+| **General Questions**                    |
+| ---------------------------------------- |
+| [Where’s my HFC?](#q0) |
+| [What are private keys and where are they stored?](#q1)  |
+| [How do I backup my private keys?](#q2)             |
+| [What happens if I lose my private keys?](#q3) |
+| [What happens if I forget my wallet passphrase?](#q4)             |
+| [How do I send HFC to other people?](#q5) |
+| [What is a Security Pic?](#q6) |
+| [My HFC balance doesn’t update instantly. Why?](#q7) |
+| [Do I get charged if a transaction fails?](#q8)   |
+| [How do I convert HFC to other currencies?](#q9)            |
+| [Who can I reach out to for additional questions?](#q10) |
+
+<a id="q0"></a>
+<br>
+
+
+###Commerce Questions
+<br>
+<a id="q0"></a>
+**Where’s my HFC?**<br> 
+High Fidelity commerce is in open beta right now. Want more HFC? Get it by going to [Bank Of High Fidelity](https://hifi.place/BankOfHighFidelity) and meeting with the banker!
+<p align="right"><a href="#top">Back to Top</a></p>
+<br>
+
+<a id="q1"></a>
+**What are private keys and where are they stored?**<br> 
+A private key is a secret piece of text that is used to prove ownership, unlock confidential information, and sign transactions. In High Fidelity, your private key is used to securely access the contents of your Wallet and Purchases. After wallet setup, a hifikey file is stored on your computer in High Fidelity Interface's AppData directory. Your hifikey file contains your private key and is protected by your wallet passphrase.
+It is very important to backup your hifikey file! To find the folder where your hifikey file is stored, navigate to the Help section in Interface. ![](HiFiPrivateKeyStorage.PNG)
```

Comment:
"hifikey" <-- that looks funny, is this a typo?  hifi-key?

Reply:
.hifikey is the file type